### PR TITLE
Feat/refactoring summery list

### PR DESCRIPTION
**Pull request by g-m-taylor**
#### Tests
Does this PR include tests for the changes introduced?
- [x] Yes
- [ ] No, I found it difficult to test
- [ ] No, they are not required for this change

### CODE REVIEW

File: src/app/shared/components/staff-record-summary/employment/employment.component.html [hunk 1]
```diff
@@ -25,14 +23,12 @@ <h2 class="govuk-heading-m">Employment details</h2>
...
-          Provide information
-          <span class="govuk-visually-hidden"> for</span>
-        </ng-container>
-        <ng-container *ngIf="worker.mainJobStartDate"> Change </ng-container>
-        <span class="govuk-visually-hidden"> job start date</span>
-      </a>
+      <app-summary-record-change
+        [explanationText]="' job start date'"
+        [link]="getRoutePath('main-job-start-date')"
+        [hasData]="!!worker.mainJobStartDate"
```
Comment (joannafawl) [inline src/app/shared/components/staff-record-summary/employment/employment.component.html:29]: How come you're using double exclamation marks?

File: src/app/shared/components/staff-record-summary/qualifications-and-training/qualifications-and-training.component.html [hunk 2]
```diff
@@ -103,16 +87,12 @@
       </app-summary-record-value>
     </dd>
     <dd class="govuk-summary-list__actions">
```
Comment (joannafawl) [inline src/app/shared/components/staff-record-summary/qualifications-and-training/qualifications-and-training.component.html:89]:
```suggestion
    <dd *ngIf="canEditWorker" class="govuk-summary-list__actions">
```
Do we need to add the `canEditWorker` permission here?

File: src/app/shared/components/staff-record-summary/employment/employment.component.html (hunk 1 above)
Comment (g-m-taylor) [reply src/app/shared/components/staff-record-summary/employment/employment.component.html:29]: !! will return a boolean (of if the value exists), so hasData doesnt care what the mainJobStartDate is , it just cares about if there is one or not.

**Review by joannafawl — APPROVED** [review]

File: src/app/shared/components/staff-record-summary/qualifications-and-training/qualifications-and-training.component.html (hunk 2 above)
Comment (g-m-taylor) [reply src/app/shared/components/staff-record-summary/qualifications-and-training/qualifications-and-training.component.html]: Good catch, it should be in the dd not the a.

**Pull request merged**